application.py

```diff
@@ -69,6 +73,15 @@ def check_for_maintenance():
             return render_template("error/maintenance.html"), 503
         elif not session['admin']:
             return render_template("error/maintenance.html"), 503
```

Comment:
this section is causing test_maintenance and later test to fail, can you take a look

Reply:
That section was created during resolution of merge conflicts. I thought it was already like that in master?

Reply:
It seems to have changed after I forked the project. I have gotten rid of it.